src/main/resources/templates/receiverTemplate.hbs

```diff
@@ -11,18 +18,21 @@
             <Exits>
                 {{#each exits}}
                     {{#with this as |Response|}}
-                        {{#when Response.statuscode 'eq' '200'}}<Exit path="Success" state="SUCCESS" code="200" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
-                        {{#when Response.statuscode 'eq' '201'}}<Exit path="Created" state="SUCCESS" code="201" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
-                        {{#when Response.statuscode 'eq' '202'}}<Exit path="Accepted" state="SUCCESS" code="202" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
-                        {{#when Response.statuscode 'eq' '204'}}<Exit path="NoContent" state="SUCCESS" code="204" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
-                        {{#when Response.statuscode 'eq' '301'}}<Exit path="MovedPermanently" state="SUCCESS" code="301" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
-                        {{#when Response.statuscode 'eq' '304'}}<Exit path="NotModified" state="SUCCESS" code="304" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
-                        {{#when Response.statuscode 'eq' '400'}}<Exit path="BadRequest" state="ERROR" code="400" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
-                        {{#when Response.statuscode 'eq' '401'}}<Exit path="NotAuthorized" state="ERROR" code="401" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
-                        {{#when Response.statuscode 'eq' '403'}}<Exit path="NotAllowed" state="ERROR" code="403" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
-                        {{#when Response.statuscode 'eq' '412'}}<Exit path="PreconditionFailed" state="ERROR" code="412" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
-                        {{#when Response.statuscode 'eq' '500'}}<Exit path="ServerError" state="ERROR" code="500" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
-                        {{#when Response.statuscode 'eq' '504'}}<Exit path="Timeout" state="ERROR" code="504" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />{{/when}}
+                        {{!-- Success Status Codes --}}
+                        {{> statusExit code="200" path="Success" state="SUCCESS"}}
```

Comment:
Er zijn ontzettend veel status codes die bij een API gebruikt kunnen worden. Ik mis hier bijvoorbeeld 404 Not Found, maar er zijn nog veel andere minder populaire status codes. Kan je een manier verzinnen hoe deze lijst generiek gemaakt kan worden, zodat het alle status codes kan ondersteunen?